src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -62,11 +62,13 @@ export class DefaultSamLocalInvokeCommand implements SamLocalInvokeCommand {
                     this.emitMessage(text)
                     // If we have a timeout (as we do on debug) refresh the timeout as we receive text
                     params.timeout?.refresh()
+                    this.logger.verbose(`stdout: ${text}`)
```

Comment:
i think `this.emitMessage()` needs to route this . or we need to route it some other way. I see this in the logs:

```
2020-10-23 22:22:14 [VERBOSE]: stderr: .
2020-10-23 22:22:14 [VERBOSE]: stderr: .
2020-10-23 22:22:15 [VERBOSE]: stderr: .
2020-10-23 22:22:15 [VERBOSE]: stderr: .
2020-10-23 22:22:15 [VERBOSE]: stderr: ...
2020-10-23 22:22:15 [VERBOSE]: stderr: .
2020-10-23 22:22:15 [VERBOSE]: stderr: .
```

but I don't see why we wouldn't instead route it to the channel+log handler as before.